recipe/meta.yaml

```diff
@@ -8,24 +8,24 @@ source:
   sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
 
 build:
-  number: 0 
+  number: 1
   script: pip install --no-deps .
 
 requirements:
   build:
     - python
     - pip
     - zlib 1.2*
-    - libtiff 4*
-    - jpeg 8*
+    - libtiff 4.0.6
+    - jpeg 9*
     - freetype 2.5*
     - tk
   run:
     - python
-    - jpeg 8*
+    - jpeg 9*
     - zlib 1.2*
     - freetype 2.5*
-    - libtiff 4*
+    - libtiff 4.0.6
```

Comment:
> or there is a chance of getting the default channel libtiff 4.0.2 that is compiled against jpeg 8d.

I don't understand that... the defaults libtiff looks like it is pinned properly, and shouldn't allow a jpeg 8d installation...

```
$ conda execute list_deps.py defaults --packages libtiff --dependencies jpeg
Fetching package metadata: ....
libtiff 4.0.2 1 None               : jpeg 8d
libtiff 4.0.6 0 None               : jpeg 8d
libtiff 4.0.6 1 None               : jpeg 8d
```